src/internal/client.ts

```diff
@@ -953,14 +953,15 @@ export class TypedClient {
     bucketName: string,
     objectName: string,
     getOpts: VersionIdentificator = {},
+    metaData: ItemBucketMetadata = {},
```

Comment:
the option for SSE should be part of getOpts. you can update the type and add the SSE fields.

Reply:
Hi @prakashsvmx , I've made the changes to include SSE headers in getOpts. Tell me if it's better

Reply:
Yes. Please resolve the conflicts. :+1:

Reply:
Here it is @prakashsvmx. Conflicts resolved 👍